tests/core/test_graph.py

```diff
@@ -813,6 +813,35 @@ def f():
     allocation_benchmark(f)
 
 
+@pytest.mark.benchmark(group="StellarGraph creation (peak)", timer=peak)
+# various element counts, to give an indication of the relationship
+# between those and memory use (0,0 gives the overhead of the
+# StellarGraph object itself, without any data)
+@pytest.mark.parametrize("num_nodes,num_edges", [(0, 0), (100, 200), (1000, 5000)])
+# features or not, to capture their cost
+@pytest.mark.parametrize("feature_size", [None, 100])
+@pytest.mark.parametrize("force_adj_lists", [None, "directed", "undirected", "both"])
+def test_allocation_benchmark_creation_peak(
```

Comment:
This function and `test_allocation_benchmark_creation` above could both call a function with the same code instead of repeating it?

Reply:
Good point; it's a bit fiddly so I'll do it as a separate commit.